packages/vats/test/bootstrapTests/test-vaults-integration.js

```diff
@@ -69,6 +69,110 @@ test.after.always(t => {
+    'prioritySenders manager',
+    // TODO? move to provisioning vat?
+    'clientCreator',
+    // in registerNetworkProtocols(). TODO: outboard #7044
+    'ProtocolHandler', // in makeLoopbackProtocolHandler()
+    'callbacks',
+    'listener',
+    // TODO(#7563): disable stakeFactory in test-vaults-config
+    'stakeReporter',
+    // in startWalletFactory()
+    // TODO(#5885): vbank should provide a facet attenuated
+    // to only provide getAssetSubscription
+    // meanwhile, expose the whole poolBank rather than
+    // export this Far object from bootstrap?
+    'AssetPublisher',
+    // in addBankAssets()
+    // XXX is attenuation needed here?
+    'AssetHub',
+    // XXX price-feed-proposal uses makeIssuerKit
+    'USD brand',
+    'ATOM brand',
+    // from makeAddressNameHubs(),
+    // makeMyAddressNameAdminKit(),
+    // makeAgoricNamesAccess()
+    'nameHub',
+    'nameAdmin',
+  ];
+
+  const { controller } = t.context;
+  const kState = controller.dump();
+  const bootstrapExports = kState.kernelTable.filter(
+    o => o[1] === 'v1' && o[2].startsWith('o+'),
+  );
+  const v1VatTable = kState.vatTables[0];
+  t.is(v1VatTable.vatID, 'v1');
+  const { transcript } = v1VatTable.state;
+
+  const oids = new Set(bootstrapExports.map(o => o[2]));
+  const oidsDurable = [...oids].filter(o => o.startsWith('o+d'));
+  t.log(
+    'bootstrap exports:',
+    oidsDurable.length,
+    'durable',
+    oids.size,
+    'total',
+  );
+  t.true(oids.size <= 15);
+  t.true(oids.size - oidsDurable.length <= 13);
+
+  // Map oid to iface by poring over transcript syscalls
+  const toIface = new Map();
+  const anObj = Far('obj', {});
+  const aPromise = harden(new Promise(() => {}));
+  const saveBootstrapIface = (slot, iface) => {
+    if (slot.startsWith('p')) return aPromise;
+    if (oids.has(slot)) {
+      toIface.set(slot, iface);
+    }
+    return anObj;
+  };
+  const m = makeMarshal(undefined, saveBootstrapIface);
+  oids.forEach(oid => {
+    for (const [_ix, ev] of transcript) {
+      for (const sc of ev.sc) {
+        if (sc.s[0] === 'send') {
+          const { methargs } = sc.s[2];
+          if (!methargs.slots.includes(oid)) continue;
+          m.fromCapData(methargs);
+          return;
+        } else if (sc.s[0] === 'resolve') {
+          for (const res of sc.s[1]) {
+            const capdata = res[2];
+            if (!capdata.slots.includes(oid)) continue;
+            m.fromCapData(capdata);
+            return;
+          }
+        }
+      }
+    }
+  });
+
+  const exportedInterfaces = [...toIface.values()].map(iface =>
+    iface.replace(/^Alleged: /, ''),
+  );
+
+  // ava does a poor job of diffing string arrays
+  const extra = setDiff(exportedInterfaces, expectedIfaces);
+  t.deepEqual(extra, [], 'unexpected exported interfaces');
+  const missing = setDiff(expectedIfaces, exportedInterfaces);
+  t.deepEqual(missing, [], 'missing exported interfaces');
```

Comment:
An alternative might be to make them a record with a `true` value or something. But yeah unfortunate.

Reply:
That seems worthwhile to get a single assertion.

Reply:
done in fixup!